data/unpacked/latest.json

```diff
@@ -774,62 +718,6 @@
         1403816400000,
         1406844000000,
         1411678800000,
-        1429826400000,
```

Comment:
I also included 2015d which removed DST in Egypt. http://mm.icann.org/pipermail/tz-announce/2015-April/000031.html